--- lightdm-gtk-greeter-1.8.4/data/lightdm-gtk-greeter.conf~	2014-03-29 09:23:56.000000000 +0000
+++ lightdm-gtk-greeter-1.8.4/data/lightdm-gtk-greeter.conf	2014-04-09 15:23:10.256516264 +0000
@@ -16,9 +16,9 @@
 # screensaver-timeout = Timeout (in seconds) until the screen blanks when the greeter is called as lockscreen
 # 
 [greeter]
-#background=
+background=/usr/share/mdk/backgrounds/default.png
 #theme-name=
-#icon-theme-name=
+icon-theme-name=gnome
 #font-name=
 #xft-antialias=
 #xft-dpi=
